src/Cms/PageActions.php

```diff
@@ -366,7 +366,19 @@ public function changeTitle(
 		string $title,
 		string|null $languageCode = null
 	): static {
+		// if the `$languageCode` argument is not set and is not the default language
+		// the `$languageCode` argument is sent as the current language
+		if (
+			$languageCode === null &&
+			$language = $this->kirby()->language()
+		) {
+			if ($language->isDefault() === false) {
+				$languageCode = $language->code();
+			}
+		}
```

Comment:
One liner if you want:

```php
$languageCode ??= $this->kirby()->language()?->isDefault() === false ? $this->kirby()->language()->code() : null;
````

Reply:
Is it needed here to check for the default language? Couldn't we just always send the code in multi-language mode?

Reply:
I already thought like that but we had implemented like in the pr: https://github.com/getkirby/kirby/blob/4.0.3/src/Cms/PageActions.php#L161

Reply:
I already thought like that but we had implemented like in the pr: https://github.com/getkirby/kirby/blob/4.0.3/src/Cms/PageActions.php#L161

Reply:
I'm not sure I understand what you mean

Reply:
Language code passed as null only when not multi-lang or default language for `changeSlug` action.